x-pack/plugins/apm/server/lib/errors/get_error_groups.ts

```diff
@@ -37,7 +36,10 @@ export async function getErrorGroups({
 }) {
   const { start, end, uiFiltersES, client, config } = setup;
 
-  const params: SearchParams = {
+  // sort buckets by last occurrence of error
```

Comment:
TS was complaining about changing the shape of the object after creating it, so I moved it inline.